### DISCUSSION TITLE
So how do I share my MAUI.Blazor

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#5070] [General]
Do I publish it to windows store?

**Comment by Xyncgas** [comment]
Any updates?

**Reply by jfversluis** [reply]
Check out this link for more information #4329 

Or this video I made about that: https://www.youtube.com/watch?v=FNwv_W3TtSU

**Reply by Xyncgas** [reply]
Windows phone should've released after MAUI, people can just publish their app to windows store, and install it on windows phone, that has a modern OS with beautiful and complete layout / control but no visible grids like those squares tiles that sits there like a brick, and we will have it easier developing for windows app store than for apple store and google play store and our app is immediately targeting all windows devices mobile or laptop or desktop

**Reply by Xyncgas** [reply]
I mean sqaure looked nice
